hilti/runtime/include/types/bytes.h

```diff
@@ -234,9 +234,11 @@ class Bytes : protected std::string {
 
     /** Returns an iterator representing the first byte of the instance. */
     const_iterator begin() const { return const_iterator(0U, getControl()); }
+    const_iterator cbegin() const { return const_iterator(0U, getControl()); }
```

Comment:
It's so that the codeine for our generic `begin`/`end` can issue `cbegin/cend`, which is needed for other types than bytes. Otherwise the codegen would need to know how to treat each type, which doesn't feel better. I'll add comments.